fusor-ember-cli/app/templates/review/installation.hbs

```diff
@@ -265,53 +265,37 @@
                               routeName='subscriptions.credentials'
                               isRequired=true
                               value=model.manifest_file }}
-
-                {{#each reviewSubscriptions as |sub|}}
-                    {{review-link label='Subscription Name'
-                                  routeName='subscriptions.select-subscriptions'
-                                  value=sub.product_name }}
-                    {{review-link label='Contract Number'
-                                  routeName='subscriptions.select-subscriptions'
-                                  value=sub.contract_number }}
-                    {{review-link label='Quantity'
-                                  routeName='subscriptions.select-subscriptions'
-                                  value=sub.quantity_attached }}
-                {{else}}
-                      {{review-link label='Quantity'
-                                    routeName='subscriptions.select-subscriptions'
-                                    value="0 - no subscriptions in manifest" }}
-                {{/each}}
-
-            {{/accordion-item}}
-
-        {{else}}
-            {{#accordion-item name='Subscriptions' isOpen=isSubscriptionsOpen}}
+            {{else}}
                 {{review-link label='Subscription Management Application'
                               routeName='subscriptions.management-application' isRequired=true
                               value=deploymentController.managementApplicationName }}
-
-                {{#each reviewSubscriptions as |sub|}}
-                    {{review-link label='Subscription Name'
-                                  routeName='subscriptions.select-subscriptions'
-                                  value=sub.product_name }}
-                    {{review-link label='Contract Number'
-                                  routeName='subscriptions.select-subscriptions'
-                                  value=sub.contract_number }}
-                    {{review-link label=qtyLabel
+            {{/if}}
+
+            {{#each reviewSubscriptions as |sub|}}
+                {{review-link label='Subscription Name'
+                              routeName='subscriptions.select-subscriptions'
+                              value=sub.product_name }}
+                {{review-link label='Contract Number'
+                              routeName='subscriptions.select-subscriptions'
+                              value=sub.contract_number }}
+                {{review-link label='Quantity'
```

Comment:
@jmrodri, all looks good. I think I would change this label to `Quantity Attached` to differentiate from `Quantity Added`